packages/form/addon/components/cf-field/input/file.js

```diff
@@ -4,64 +4,92 @@ import Component from "@glimmer/component";
 import { queryManager } from "ember-apollo-client";
 import fetch from "fetch";
 
-import removeAnswerMutation from "@projectcaluma/ember-form/gql/mutations/remove-answer.graphql";
-import getFileAnswerInfoQuery from "@projectcaluma/ember-form/gql/queries/fileanswer-info.graphql";
+import getFilesAnswerInfoQuery from "@projectcaluma/ember-form/gql/queries/filesanswer-info.graphql";
 
 export default class CfFieldInputFileComponent extends Component {
   @service intl;
 
   @queryManager apollo;
 
-  get downloadUrl() {
-    return this.args.field?.answer?.value?.downloadUrl;
+  get files() {
+    return this.args.field?.answer?.value;
   }
 
-  get downloadName() {
-    return this.args.field?.answer?.value?.name;
+  get allowedMimeTypes() {
+    // TODO: put this into a config env value
+    return ["image/png", "image/jpeg", "text/plain", "application/pdf"];
```

Comment:
This looks unfinished :-) Also, is there a reason we're "sneaking in" this new  feature here?